docs/DeveloperGuide.md

```diff
@@ -175,34 +175,22 @@ This section describes some noteworthy details on how certain features are imple
 The add course mechanism is facilitated by `AddCommand`. It extends `Command` which overrides the following operation:
 * `AddCommand#execute():` Adds a person into NUSCoursemates
 
-Given below is an example usage scenario and how the add mechanism behaves at each step.
+Given below is an example usage scenario and how the add mechanism behaves at each step focusing on the `Course` field.
 
-Step 1. The user will input the add command along with the person's name and the course that the person is taking.
+Step 1. The user executes `add n/John p/81234567 \n c/CS2103T` to add a new person.
```

Comment:
Great command! But I wonder if the `\n` here is part of the command or for formatting purposes.